crates/core/Cargo.toml

```diff
@@ -31,6 +31,7 @@ pest = "2.0"
 pest_derive = "2.0"
 
 # Exposed in API
+regex = "1.5"
```

Comment:
This dependency was added in the "public dependencies" section.  I'm not aware of this being a public dependency though

Reply:
Whoops, thanks... fixing that now.

Reply:
Moved into private area.